docs/Resources-Appendices/operators.md

```diff
@@ -39,22 +43,22 @@ z = x / ++x;
 Programs should not depend upon the order of evaluation of expressions, except as guaranteed by ANSI C for the following operators:
 
 ```
```

Comment:
Based off the checklist provided in #18, 
- [ ] Make sure all code blocks use syntax highlighting
This could utilize a syntax highlight such as c, or c++.

Reply:
I think this is OK, since this isn't really C code, just C code in text.